Homework_Conditions/index.js

```diff
@@ -77,19 +78,22 @@ function dressListInput() {
                 document.getElementById("selectDress").innerHTML = "Out of Stock";
             }
         });
-        /* How can i add stars here to the property rating */
-        {
-            /* <span class="fa fa-star unchecked" id=”two”></span>
-                    <span class="fa fa-star unchecked" id=”three”></span>
-                    <span class="fa fa-star unchecked" id=”four”></span> */
-        }
-        /* I want to add stars like this to my property rating */
-        function createStars() {
-
 
-        }
+        function createStars(starCount) {
```

Comment:
you need to take this function out of `dressListInput` function, because when you loop through the array on each step you recreate this function. The usage is correct and would stay the same.

Reply:
resolved this

Reply:
added buttons to the dress card and the amount means the amount of dress that user wants before adding to cart.Not able figure about the duplication